cargo-verify/src/main.rs

```diff
@@ -97,6 +97,15 @@ pub struct Opt {
     #[structopt(long, value_name = "PATH", env = "SEAHORN_VERIFY_C_COMMON_DIR")]
     seahorn_verify_c_common_dir: Option<String>,
 
+    /// Which LLVM version to use (e.g., 10 or 11)
```

Comment:
The question is: how to bake the version in?
This allows it to be set by an environment variable (that is set by the docker container) or to be set by a command line.
I think this is pretty flexible and convenient.

(The other alternatives I can think of are to generate a .rs file that is used at build time to bake the version in or to generate a configuration file that is read when cargo-verify starts. I think both of those are less convenient.)

Reply:
No, this runtime switch does not effect the build of rvt-patch-llvm or the build of the code in runtime/src (you forgot to change the Makefile in the latter? it uses clang-10), if you build those with llvm-11 and then run cargo-verify with llvm-10 you will probably run into trouble.

Reply:
Don't forget to

1) run docker/init which uses LLVM_VERSION when rebuilding rvt_patch_llvm
2) make an LLVM11-based rustc your default in rustup. I am using nightly-2021-02-20-x86_64-unknown-linux-gnu (which is what Rust-for-Linux depends on).

If you do this, I believe that everything will be consistent.
Please let me know if I missed anything.

Reply:
You can indeed get things to be consistent, but you're making it unnecessarily easy to get things inconsistent. LLVM version is not a runtime option for any other tool (external or internal to the project) and it shouldn't be for cargo-verify.

Reply:
I think this PR works and I have not seen problems.
Since we are currently only recommending LLVM10 at the moment and that is the default, it should not cause problems for users either.
If there is an easy improvement we could make, I'm open to suggestions - but let's not overcomplicate this.